README.md

```diff
@@ -1,38 +1,74 @@
 # IWD dev team ladder
 
+This job ladder exposes all roles inside IWD development department, and define the goals, expectations and responsibilities of each role.
```

Comment:
IWD development department --> IWD dev team

I feel less big corp and more welcoming for new comer

Reply:
fixed